source/includes/_ais.md

```diff
@@ -325,6 +325,25 @@ or `ReadAccountsDetailed` permission. In the former case, we will omit the Pot n
 We'll only return open pots as part of our response. If a customer closes a pot, it won't appear in the response
 any more.
 
+##### Field Descriptions
+| Field name           | Mandatory | Description                                                                   | Type                                  |
```

Comment:
`Mandatory` doesn't quite feel like the right word.. I would reverse this to be `Optional` and change the ticks/crosses